src/llm/servable_initializer.cpp

```diff
@@ -117,36 +117,107 @@ void GenAiServableInitializer::loadTextProcessor(std::shared_ptr<GenAiServablePr
     }
 }
 
-Status GenAiServableInitializer::parseModelsPath(std::string modelsPath, std::string graphPath) {
+Status parseModelsPath(std::string& outPath, std::string modelsPath, std::string graphPath) {
```

Comment:
if its not class method, but local helper, use static 

the same below in another method

Reply:
This method is also used outside of main servable initializer (namely in derived classes that include this header).
Below method determinePipelineType has dedicated tests specifically checking it's correctness. They are not used only in the scope of basic servable initializer.